globus_action_provider_tools/flask/helpers.py

```diff
@@ -0,0 +1,48 @@
+from enum import Enum
+from typing import Iterable, List, Set
+
+from flask import request
+
+
+def parse_query_args(
+    arg_name: str, default_value: str = "", valid_vals: List[str] = None
```

Comment:
`valid_vals` as a `Set` seems to make sense esp since this is returning a `Set`.